HealthcareSystem/Backend/Migrations/20201127131042_NMGE.cs

```diff
@@ -0,0 +1,166 @@
+﻿using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Backend.Migrations
+{
+    public partial class NMGE : Migration
```

Comment:
think out more meaningful name than NMGE

Reply:
A tip for naming things:
1) Describe the thing in a sentence or two (easier to do in Serbian).
2) Remove any words that are not crucial to convey the meaning.
3) Remove any words that can be inferred from the Type, containing function, containing class, or containing module.
4) See if what remains sufficiently describes the thing.
5) Rename.